src/store/concepts/lists/operations/loadLists.js

```diff
@@ -0,0 +1,44 @@
+import { createLogic } from 'redux-logic'
+
+import { userIdSelector } from 'Store/concepts/account/selectors'
+import { dataApiSave, dataApiSuccess } from 'Store/concepts/data/actions'
+import storage from 'Utils/storage'
+import normalizeLists from 'Store/schemas/lists'
+
+import { LOAD_LISTS } from '../types'
+import { getListsEndpoint, lists } from '../endpoints'
+import { saveListsIds, saveTotal } from '../actions'
+
+const loadListsOperation = createLogic({
+  type: LOAD_LISTS,
+  latest: true,
+  async process(
+    {
+      httpClient,
+      action: {
+        page
+      },
+      getState
+    },
+    dispatch,
+    done
+  ) {
+    const userId = userIdSelector(getState())
+    const {
+      data: {
+        results,
+        total_results: totalResults
+      }
+    } = await httpClient.get(getListsEndpoint(userId), { params: { page, session_id: storage.session.get() } })
+    const normalizedData = normalizeLists(results)
+
+    dispatch(dataApiSuccess({ endpoint: lists }))
+    dispatch(dataApiSave({ endpoint: lists, response: normalizedData.entities.list || {} }))
```

Comment:
use default value somewhere inside reducer instead of `|| {}`

Reply:
Needs to discuss